JSDemos/Demos/PivotGrid/OLAPDataSource/description.md

```diff
@@ -1 +1,20 @@
-The PivotGrid supports OLAP services (MS&nbsp;SQL Server Analysis Services). This allows you to&nbsp;deliver solutions that can easily analyze millions of&nbsp;data rows within your app. In&nbsp;this demo, the PivotGrid uses **XmlaStore** to&nbsp;connect the component to&nbsp;a&nbsp;sample remote OLAP service. The Field Chooser is&nbsp;configured to&nbsp;show the search box to&nbsp;speed up&nbsp;choosing the required field.
\ No newline at end of file
+The PivotGrid component supports OLAP services (Microsoft SQL Server Analysis Services). This demo shows how to use a remote OLAP cube as the PivotGrid's data source.
+
+## Configure the Store
+To connect your component to an OLAP storage, use [XmlaStore](/Documentation/ApiReference/Data_Layer/XmlaStore/). Specify the following properties to configure it:    
```

Comment:
[XmlaStore](/Documentation/ApiReference/Data_Layer/XmlaStore/) My first question - is it a component? Why don't you name it as such? Yeah, I understand that you'll need to change the beginning of the sentence as well. For instance:

> To connect the PivotGrid to an OLAP storage, use the `XmlaStore` component.

Second question - Do the listed properties belong to the mentioned `XmlaStore`? Looks like they do, therefore, you need to show this connection clearer. The easiest way to do it is to include `XmlaStore` into property names as a path. For example:

> [XmlaStore.url](/Documentation/ApiReference/Data_Layer/XmlaStore/Configuration/#url)

Third question. Do you always list properties like this? I mean that you start a list item with a property name followed by a sentence without punctuation. I would have enclosed text in brackets. However, if you always do like this, I have no objections.

Reply:
1. Hello. We usually don't refer to the stores of different types as components. They are called just stores.
2. Why use so much repetitions if these properties belong to the same store? We can show this connection as shown in [this demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/SelectBox/Overview/) instead.
3. Yes, this is a common way for us.

Reply:
Alright, I see.
> We can show this connection as shown in this demo instead.

Looks like a good idea.